mem/basic.md

```diff
@@ -204,7 +204,7 @@ impl BasicMemory {
             throw_ub!("out-of-bounds memory access");
         }
         // All is good!
-        ret(Some((id, Size::from_bytes(offset_in_alloc))))
+        ret(Some((id, Size::from_bytes(offset_in_alloc).unwrap())))
```

Comment:
A few lines above, we asserted that offset_in_loc is not negative, so this unwrap should be safe too.